webapp/src/app/pacman-features/modules/compliance/card/card.component.ts

```diff
@@ -63,6 +63,8 @@ export class CardComponent implements OnInit {
   
 
   handleGraphIntervalSelection = (e) => {
+    this.fromDate = new Date(2022, 1, 1);
+    this.toDate = new Date(2200, 12, 31);
```

Comment:
Shouldn't we remove this hard coded dates?

Reply:
Let's go with this for now? I have noted it down.

Reply:
lets revisit this later.